tests/fixtures/markdown/code.md

```diff
@@ -0,0 +1,8 @@
+Here is some Python:
+
+```python
```

Comment:
That's probably not really the responsibility of the readme_renderer tests to validate this, but it may not hurt to add it so we're also ensuring that this particular tag, probably the most used in this context, still work, so I added it.